backend/src/test/java/gov/cdc/usds/simplereport/db/repository/TestEventRepositoryTest.java

```diff
@@ -82,13 +83,13 @@ void testFindByPatient() {
     _resultRepo.save(positiveResult);
     HashSet positiveResults = new HashSet<>();
     positiveResults.add(positiveResult);
-    _repo.save(new TestEvent(order, false, positiveResults));
```

Comment:
This is misleading, it doesnt save the `positiveResults` results to the db. Only way to persist a Result entity is via the `resultsRepository.save()`